### DISCUSSION TITLE
安装master分支，在启动FE时出现问题

### DISCUSSION THREAD

**Discussion by KeithKaysville** [apache/doris#6610] [A - General / Q&A] [answered]
下面是我在启动时 出现的日志问题
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/root/incubator-doris-master/output/fe/lib/slf4j-log4j12-1.7.28.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/root/incubator-doris-master/output/fe/lib/log4j-slf4j-impl-2.14.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
java.io.IOException: the self host 172.17.0.1 does not equal to the host in ROLE file 172.17.0.3. You need to set 'priority_networks' config in fe.conf to match the host 172.17.0.3
        at org.apache.doris.catalog.Catalog.getClusterIdAndRole(Catalog.java:888)
        at org.apache.doris.catalog.Catalog.initialize(Catalog.java:790)
        at org.apache.doris.PaloFe.start(PaloFe.java:121)
        at org.apache.doris.PaloFe.main(PaloFe.java:62)

**Comment by crazy2323** [comment, marked as answer]
fe.conf里面的priority_networks没设置对

**Comment by KeithKaysville** [comment]
您好，但是我这里已经在fe.conf 中配置了ID，但是好像配置并未生效




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/incubator-doris"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月10日(星期五) 下午3:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/incubator-doris] 安装master分支，在启动FE时出现问题 (Discussion #6610)





 
fe.conf里面的priority_networks没设置对
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by crazy2323** [reply]
ip addr把内网的网卡ip找出来，修改fe.conf文件，priority_networks=内网ip/16

**Reply by coderjiang** [reply]
我也碰到这个问题，如何解决啊，求救。。。

**Reply by JunDaTang** [reply]
你设置后也不生效对吧？ 我也是，我就直接删除doris整目录，从源镜像直接重新解压一份doris镜像进行配置就解决了。（ps:我的doris从别的节点直接复制过来，可能被运行过，某处文件还保存着原节点的ip)

**Reply by shenghongda** [reply]
你是在虚拟机中运行的吗？

**Comment by 804e** [comment]
清理掉doris-meta目录

**Reply by Smelly-calf** [reply]
这个解决我的问题了，🐮

**Reply by okay456okay** [reply]
先删除doris-meta所有内容，再新建
rm -rf doris-meta
mkdir doris-meta
